web/src/recent_topics_ui.js

```diff
@@ -743,6 +743,7 @@ function topic_offset_to_visible_area(topic_row) {
 }
 
 function set_focus_to_element_in_center() {
+    popovers.hide_all();
```

Comment:
This adds a change that is definitely not changing the focus to a function with a clear name; we should call this function in the callback, but not this way.